class1/exercises.livemd

```diff
@@ -95,8 +116,24 @@ Tip: You can calculate score for each group in separate function and pipe them t
 
 ```elixir
 defmodule Scrabble do
+  defp get_letter_value(letter) do
+    cond do
+      "AEIOULNRST" =~ letter -> 1
+      "DG" =~ letter -> 2
+      "BCMP" =~ letter -> 3
+      "FHVWY" =~ letter -> 4
+      "K" =~ letter -> 5
+      "JX" =~ letter -> 8
+      "QZ" =~ letter -> 10
+    end
+  end
+
   def calculate_score(word) do
-    # Your code here
+    word
+    |> String.upcase()
+    |> String.graphemes()
+    |> Enum.map(fn x -> get_letter_value(x) end)
```

Comment:
Very nice solution ;) Correct usage of pipe operator.
Small tip: if you are are only sending enum item to the function as is, you can use shorter notation:
`|> Enum.map(&get_letter_value/1)`

Reply:
That will come in handy.
This implementation with reduced sugar syntax is more appealing to me.